### PR TITLE
Minor fixes to recording logging and one change in logging.

### PR DESCRIPTION
**Pull request by hjdhjd**
## :recycle: Current situation

Two fixes for logging:

- A missing connection identifier in the debug logging of the close handler.
- Message notifying users that isLast is missing used string literals with the message broken up over two lines for code formatting purposes. I've adjusted it to output as-intended on a single line.

Removed one log warning related to not sending any data out through the recording handler. There are valid situations where not sending out any data is the correct answer such as when the HomeKit hub specifically terminates the connection before the first recording packet is sent out. In that instance, sending any data back would result in a different error (Delegate yielded fragment after close). In truth, while this should be an edge case that rarely occurs, it does occur regularly enough that you'll see it in a typical home environment when you have more than a few cameras that are producing HomeKit Secure Video events.
 
## :bulb: Proposed solution

See above.

## :gear: Release Notes

Minor logging improvements for HomeKit Secure Video recording delegates.

## :heavy_plus_sign: Additional Information
*If applicable, provide additional context in this section.*

### Testing

*Which tests were added? Which existing tests were adapted/changed? Which situations are covered, and what edge cases are missing?*

### Reviewer Nudging

@donavanbecker @Supereg

### CODE REVIEW

**Review by donavanbecker — APPROVED** [review]

**Review by bauer-andreas — APPROVED** [review]

File: src/lib/camera/RecordingManagement.ts [hunk 1]
```diff
@@ -1056,8 +1056,8 @@ class CameraRecordingStream extends EventEmitter implements DataStreamProtocolHa
 
       if (!lastFragmentWasMarkedLast && !this.closed) {
         // Delegate violates the contract. Exited normally on a non-closed stream without properly setting `isLast`.
-        console.warn(`[HDS ${this.connection.remoteAddress}] Delegate finished streaming for ${this.streamId} without setting RecordingPacket.isLast. \
-        Can't notify Controller about endOfStream!`);
+        console.warn(`[HDS ${this.connection.remoteAddress}] Delegate finished streaming for ${this.streamId} without setting RecordingPacket.isLast. ` +
+        "Can't notify Controller about endOfStream!");
```
Comment (Shaquu) [inline src/lib/camera/RecordingManagement.ts:1060]: Personally I do not like it this way

Comment (hjdhjd) [reply src/lib/camera/RecordingManagement.ts:1060]: Current implementation produces:

```
[HDS ::ffff:1.2.3.4] Delegate finished streaming for 1 without setting RecordingPacket.isLast.         Can't notify Controller about endOfStream!
```
This is incorrect - having the error include spurious spaces where they are not needed nor appropriate is grammatically incorrect and confusing. 😄 

The `\` **is** the error. 

The choices are to either violate code formatting consistency and have the line be unnaturally long, change the message to be shorter, or use two lines of code. I chose the option that preserved the message and preserved the source code formatting preferences. 

If you've got a better recommendation that's consistent with the coding style / standards in HAP-NodeJS, I welcome it...sometimes you choose the best of suboptimal options, but if there's something missed, please provide an alternative that addresses the issue within the above constraints.

The code now more correctly produces:

```
[HDS ::ffff:1.2.3.4] Delegate finished streaming for 1 without setting RecordingPacket.isLast. Can't notify Controller about endOfStream!
```

Comment (donavanbecker) [reply src/lib/camera/RecordingManagement.ts:1060]: Are we good to merge and get into v0.12.3?
